Directory.Build.props

```diff
@@ -0,0 +1,18 @@
+<Project>
+  <Import Project="Sdk.props" Sdk="Microsoft.DotNet.Arcade.Sdk" />
+
+  <PropertyGroup>
+    <Copyright>$(CopyrightNetFoundation)</Copyright>
+    <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <TreatWarningsAsErrors>True</TreatWarningsAsErrors>
+    <DebugType>embedded</DebugType>
+    <DebugSymbols>true</DebugSymbols>
+    <LangVersion>Latest</LangVersion>
+    <IsPackable>true</IsPackable>
+
```

Comment:
Just a note that it might be worth it to check a project's Nuget package after it is built for 
1. Author
2. Project URL

Reply:
Yeah, these are set by arcade. Just doubled checked and it makes sense.